server/src/main/java/com/clarkparsia/pellet/server/protege/ProtegeOntologyState.java

```diff
@@ -198,6 +205,11 @@ public void save() {
 
 	@Override
 	public void close() {
+		try {
+			clearRevisionFile();
+		} catch (IOException e) {
+			LOGGER.log(Level.INFO, "Couldn't delete revision file for " + this.projectId.get() + ": " + e);
+		}
```

Comment:
IMO, you might as well shove this exception wrapping into clearRevisionFile itself.

Looks  good otherwise.